sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Security/Ucfg/UcfgBuilderTestBase.cs

```diff
@@ -48,8 +48,14 @@ protected UCFG GetUcfgForMethod(string code, string methodName)
 
             var builder = new UniversalControlFlowGraphBuilder();
 
-            return builder.Build(semanticModel, method,
-                semanticModel.GetDeclaredSymbol(method), CSharpControlFlowGraph.Create(method.Body, semanticModel));
+            var cfg = CSharpControlFlowGraph.Create(method.Body, semanticModel);
+
+            var ucfg = builder.Build(semanticModel, method, semanticModel.GetDeclaredSymbol(method), cfg);
+
+            var serializedCfg = CfgSerializer.Serialize(methodName, cfg);
+            var serualizedUcfg = UcfgSerializer.Serialize(ucfg);
+
+            return ucfg;
```

Comment:
put a breakpoint and use http://www.webgraphviz.com/ to see what's the shape of the graphs